config/src/config/mempool_config.rs

```diff
@@ -85,8 +85,8 @@ impl Default for MempoolConfig {
             system_transaction_timeout_secs: 600,
             system_transaction_gc_interval_ms: 60_000,
             broadcast_buckets: DEFAULT_BUCKETS.to_vec(),
-            eager_expire_threshold_ms: Some(10_000),
-            eager_expire_time_ms: 3_000,
+            eager_expire_threshold_ms: Some(15_000),
```

Comment:
Can you explain the downside of increasing this?

Reply:
new constants make the logic like this:

if there is any transaction in the mempool (that never entered parking lot) that was there for longer than eager_expire_threshold_ms (which is increased from 10s -> 15s here) - that means we have a significant backlog.

if that is the case, we pull into batches only transactions that have at least eager_expire_time_ms time left (increased from 3s to 6s)


We've increased both, as with reduced QS backpressure, we see it taking more time from batch creation to batch being included in the block.

So the side-consequence of this change is that - if that during backlog, if you use expiration <6s, your transactions will be ignored (previously <3s).

Reply:
Sounds good to me. Slightly orthogonal but do we really need this logic after @vusirikala 's change to exclude expired transaction in QS backpressure calculation in https://github.com/aptos-labs/aptos-core/pull/13850/files. cc - @bchocho

Reply:
creating useless batches, and then requiring everyone to fetch them, just in order to throw them out - probably has negative effect on our overall throughput.

Reply:
@sital This is the run for graceful overload test with my changes and increasing QS backpressure limits without changing the `eager_expire_threshold_ms` config.
https://github.com/aptos-labs/aptos-core/pull/13964#issuecomment-2221167236